nx-dev/nx-dev/lib/githubStars.api.ts

```diff
@@ -0,0 +1,18 @@
+import { Octokit } from 'octokit';
+
+export async function fetchGithubStarCount() {
+  try {
+    const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+    const responseData = await octokit.request('GET /repos/{owner}/{repo}', {
+      owner: 'nrwl',
```

Comment:
Let's make this generic so we can reuse it for third-party packages.
Have `owner` and `repo` be input parameters.

Reply:
Let's do that in follow-up PRs once we tackle the community plugin part